app/team.go

```diff
@@ -191,6 +310,20 @@ func (a *App) CreateTeam(c request.CTX, team *model.Team) (*model.Team, *model.A
 		}
 	}
 
+	// MM-48246 A/B test show linked boards. Create a welcome to boards linked board per user
+	if a.shouldCreateOnboardingLinkedBoard(c, team.Id) {
+		board, err2 := a.createOnboardingLinkedBoard(c, team.Id)
+
```

Comment:
nit: it's usual practice to not have a blank line between a call returning an `err` and the `if`